src/pyxplor/plot_time_series.py

```diff
@@ -8,54 +10,71 @@ def plot_time_series(input_df: pd.DataFrame,
                      output_path: str = None, 
                      super_title: str = "Time Series Analysis",
                      super_title_font: int = 14) -> None:
-    """Conducts exploratory data analysis on multiple time-series variables and generates line plot visualizations.
+    """
+    Generates line plot visualizations for multiple time-series variables in a DataFrame.
 
-    This function generates line plots for multiple time series variables specified in value_columns, 
-    each aggregated at a given frequency. It creates a single figure with subplots for each variable, 
-    allowing for easy comparison of trends, seasonality, and other time-series characteristics. 
-    The function also offers the option to save and display the resulting figure.
+    This function creates a figure with subplots for each specified time series variable,
+    aggregated at a given frequency. It allows for comparisons of trends and seasonality.
 
     Parameters
     ----------
     input_df : pd.DataFrame
-        The DataFrame containing the time-series data. Expected to have at least two columns: 
-        one for dates and others for numeric values.
+        DataFrame containing the time-series data with date and numeric columns.
         
     date_column : str
-        The name of the column in `input_df` that contains date/time information. 
-        This column should be convertible to pandas datetime format.
+        Name of the column containing date/time information in datetime format.
 
     value_columns : list
-        A list of names of the columns in `input_df` that contain the numeric values to be analyzed.
+        List of column names containing numeric values to plot as time series.
 
-    freq : {"D", "W", "M"}, optional
-        The frequency of the time-series data ('D' for daily, 'W' for weekly, etc.). Default is 'D'.
+    freq : str, optional
+        Frequency of data aggregation ('D' for daily, 'W' for weekly, etc.). Default is 'D'.
 
     figsize : tuple[int, int], optional
-        The size of the plot in the format (width, height). Default is (10, 6).
+        Size of the plot as (width, height). Default is (10, 6).
 
     output_path : str, optional
-        Path to save the plot. Defaults to the current working directory.
+        File path to save the plot. If None, the plot is not saved.
 
     super_title : str, optional
-        Super title for the entire plot. Default is "Time Series Analysis".
+        Title for the entire plot. Default is "Time Series Analysis".
 
     super_title_font : int, optional
         Font size for the super title. Default is 14.
 
     Returns
     -------
     None
```

Comment:
I believe we're supposed to return the figure and axes object.

Reply:
I will check an example